### PR TITLE
Paramedsuit resprite

### PR DESCRIPTION
**Pull request by SomeCabbage**
# Description
Resprites the Paramedic Voidsuit. Also updates the metadata of the HoP suit sprite because the credit wasn't done right.

# Media
image
image

# Changelog

:cl: SomeCabbage
- tweak: Resprited the paramedic voidsuit to make it look less saturated and hard on the eyes

### CODE REVIEW

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by DSC@Cabbage#9633 (561159087765848084)",
```
Comment (Pspritechologist) [inline Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json]:
```suggestion
    "copyright": "Made by DSC@cababge#9633 (561159087765848084)",

```
:p

Comment (DEATHB4DEFEAT) [reply Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json]: :godmode:

Comment (SomeCabbage) [reply Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json]: oh shoot you're right

Comment (SomeCabbage) [reply Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json]: aw man now I gotta go and change it across all my prs

Comment (DEATHB4DEFEAT) [reply Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json]: just make a single PR with a funny file search find and replace after they're all merged :trollface:

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**